ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -236,8 +222,8 @@ objects:
           for master in "${MASTERS[@]}"
           do
             scp -o StrictHostKeyChecking=no -o ProxyCommand="ssh -A -o StrictHostKeyChecking=no -o ServerAliveInterval=30 -W %h:%p core@${BASTION_HOST}" ${KUBE_SSH_KEY_PATH} "core@${master}":/home/core/.ssh/id_rsa
-            bastion_ssh "core@${master}" "chmod 0600 /home/core/.ssh/id_rsa"
-            bastion_ssh "core@${FIRST_MASTER}" "scp -o StrictHostKeyChecking=no /var/lib/etcd/snapshot.db core@${master}:/tmp/snapshot.db"
+            bastion_ssh "core@${master}" "sudo mkdir /root/.ssh; sudo -i cp /home/core/.ssh/id_rsa /root/.ssh/id_rsa && chmod 0600 /home/core/.ssh/id_rsa && sudo -i chmod 0600 /root/.ssh/id_rsa"
```

Comment:
Why the change to copy keys to root user.

Reply:
Snapshot is produced by etcdctl which is running as sudo. An easier solution is to chmod the snapshot so that `core` could access it.

Fixed in a8f4d1346